src/html-webpack-plugin.polyfill.ts

```diff
@@ -0,0 +1,16 @@
+import { getHooks, AlterAssetTagsData, HtmlTag } from 'html-webpack-plugin'
+import { compilation } from 'webpack'
+import Compilation = compilation.Compilation
+
+const isV4 = typeof getHooks !== undefined
```

Comment:
`undefined` needs to be quoted

```suggestion
const isV4 = typeof getHooks !== 'undefined'
```